code/datums/gamemodes/pod_wars/pw_manufacturing_.dm

```diff
@@ -216,10 +225,14 @@
 	category = "Clothing"
 
 
-/obj/machinery/manufacturer/mining/pod_wars
+/obj/machinery/manufacturer/mining/pod_wars/
+	var/team_num = 0
 	New()
 		available -= /datum/manufacture/jetpack
-		available += /datum/manufacture/pod_wars/jetpack
+		if(src.team_num == TEAM_NANOTRASEN)
+			available += /datum/manufacture/pod_wars/jetpack
+		if(src.team_num == TEAM_SYNDICATE)
+			available += /datum/manufacture/pod_wars/jetpack/syndicate
```

Comment:
I changed it a couple hours ago such that each team has a unique fab (`/obj/machinery/manufacturer/mining/pod_wars/nanotrasen` and `/obj/machinery/manufacturer/mining/pod_wars/syndicate`).

Reply:
Fixed!